libs/shared/data/src/lib/countries.ts

```diff
@@ -0,0 +1,998 @@
+    label: 'Dominica',
+  },
+  {
+    value: 'DO',
+    label: 'Dominican Republic',
+  },
+  {
+    value: 'EC',
+    label: 'Ecuador',
+  },
+  {
+    value: 'EG',
+    label: 'Egypt',
+  },
+  {
+    value: 'SV',
+    label: 'El Salvador',
+  },
+  {
+    value: 'GQ',
+    label: 'Equatorial Guinea',
+  },
+  {
+    value: 'ER',
+    label: 'Eritrea',
+  },
+  {
+    value: 'EE',
+    label: 'Estonia',
+  },
+  {
+    value: 'ET',
+    label: 'Ethiopia',
+  },
+  {
+    value: 'FK',
+    label: 'Falkland Islands (Malvinas)',
+  },
+  {
+    value: 'FO',
+    label: 'Faroe Islands',
+  },
+  {
+    value: 'FJ',
+    label: 'Fiji',
+  },
+  {
+    value: 'FI',
+    label: 'Finland',
+  },
+  {
+    value: 'FR',
+    label: 'France',
+  },
+  {
+    value: 'GF',
+    label: 'French Guiana',
+  },
+  {
+    value: 'PF',
+    label: 'French Polynesia',
+  },
+  {
+    value: 'TF',
+    label: 'French Southern Territories',
+  },
+  {
+    value: 'GA',
+    label: 'Gabon',
+  },
+  {
+    value: 'GM',
+    label: 'Gambia',
+  },
+  {
+    value: 'GE',
+    label: 'Georgia',
+  },
+  {
+    value: 'DE',
+    label: 'Germany',
+  },
+  {
+    value: 'GH',
+    label: 'Ghana',
+  },
+  {
+    value: 'GI',
+    label: 'Gibraltar',
+  },
+  {
+    value: 'GR',
+    label: 'Greece',
+  },
+  {
+    value: 'GL',
+    label: 'Greenland',
+  },
+  {
+    value: 'GD',
+    label: 'Grenada',
+  },
+  {
+    value: 'GP',
+    label: 'Guadeloupe',
+  },
+  {
+    value: 'GU',
+    label: 'Guam',
+  },
+  {
+    value: 'GT',
+    label: 'Guatemala',
+  },
+  {
+    value: 'GG',
+    label: 'Guernsey',
+  },
+  {
+    value: 'GN',
+    label: 'Guinea',
+  },
+  {
+    value: 'GW',
+    label: 'Guinea-Bissau',
+  },
+  {
+    value: 'GY',
+    label: 'Guyana',
+  },
+  {
+    value: 'HT',
+    label: 'Haiti',
+  },
+  {
+    value: 'HM',
+    label: 'Heard Island and McDonald Islands',
+  },
+  {
+    value: 'VA',
+    label: 'Holy See (Vatican City State)',
+  },
+  {
+    value: 'HN',
+    label: 'Honduras',
+  },
+  {
+    value: 'HK',
+    label: 'Hong Kong',
+  },
+  {
+    value: 'HU',
+    label: 'Hungary',
+  },
+  {
+    value: 'IS',
+    label: 'Iceland',
+  },
+  {
+    value: 'IN',
+    label: 'India',
+  },
+  {
+    value: 'ID',
+    label: 'Indonesia',
+  },
+  {
+    value: 'IR',
+    label: 'Iran, Islamic Republic of',
+  },
+  {
+    value: 'IQ',
+    label: 'Iraq',
+  },
+  {
+    value: 'IE',
+    label: 'Ireland',
+  },
+  {
+    value: 'IM',
+    label: 'Isle of Man',
+  },
+  {
+    value: 'IL',
+    label: 'Israel',
+  },
+  {
+    value: 'IT',
+    label: 'Italy',
+  },
+  {
+    value: 'JM',
+    label: 'Jamaica',
+  },
+  {
+    value: 'JP',
+    label: 'Japan',
+  },
+  {
+    value: 'JE',
+    label: 'Jersey',
+  },
+  {
+    value: 'JO',
+    label: 'Jordan',
+  },
+  {
+    value: 'KZ',
+    label: 'Kazakhstan',
+  },
+  {
+    value: 'KE',
+    label: 'Kenya',
+  },
+  {
+    value: 'KI',
+    label: 'Kiribati',
+  },
+  {
```

Comment:
where is Kosovo? :/

Reply:
Hahah, you just went through the whole country list :D ... good spot!

Reply:
Soooo.. which native part should I pick:
```
Republika e Kosovës  (Albanian)
Република Косово  (Serbian)
```

Reply:
Woah, tricky. Albanian is the most spoken language, so let's go for that https://en.wikipedia.org/wiki/Demographics_of_Kosovo#Languages